pkg/agent/multicast/mcast_controller.go

```diff
@@ -131,7 +131,7 @@ func (c *Controller) updateGroupMemberStatus(obj interface{}, e *mcastGroupEvent
 	newStatus := &GroupMemberStatus{
 		group:          status.group,
 		localMembers:   make(map[string]time.Time),
-		remoteMembers:  status.remoteMembers,
+		remoteMembers:  sets.NewString(status.remoteMembers.List()...),
```

Comment:
@wenyingd could you provide a few background for this issue in the PR summary?

Reply:
I would create an issue for it.